css-brochure-form/styles.css

```diff
@@ -0,0 +1,57 @@
+.poppins-regular {
+  font-family: Poppins, sans-serif;
+  font-style: normal;
+}
```

Comment:
Works, but rather than giving this class to multiple elements, I would've just applied this to the `body`, so all of it's children would inherit the `font-family`